docs/fundamentals/executor/executor-in-flow.md

```diff
@@ -1,136 +1,125 @@
 (executor-in-flow)=
 # Executors inside a Flow
 
-Executors are a good way to group your `DocumentArray` functions and logics into a class that can share `configuration` and `state` in a modular way.
-However, the main advantage of using `Executor` is to be used in a `Flow` so that it can be served, scaled and deployed.
+Executors are a way to group your `DocumentArray` functions and processing logic into a class that can share configuration and state in a modular way.
+The main advantage of `Executor`s is using them in a `Flow`, which can serve, scale, and deploy them with ease.
 
-## Special Executor attributes
+(executor-api)=
+## YAML and Python API
 
-When implementing an `Executor`, if your executor overrides `__init__`, it needs to carry `**kwargs` in the signature and call `super().__init__(**kwargs)`
-                                 
-```python
-from jina import Executor
+An Executor can be loaded from a YAML file or via Python API. 
 
+````{dropdown} executor.py
+:open:
 
-class MyExecutor(Executor):
-    def __init__(self, foo: str, bar: int, **kwargs):
-        super().__init__(**kwargs)
-        self.bar = bar
-        self.foo = foo
-```
+This is a basic Executor, defined in its own file. Notice that there are no {ref}`request bindings <executor-requests>`. These are configured via the Python and YAML syntax.
```

Comment:
I would just make it a bit clearer that the 'old' way of defining a request mapping is still valid, and also mention the advantages of doing it this way:

```suggestion
This is a basic Executor, defined in its own file. Notice that there are no {ref}`request bindings <executor-requests>`. If an Executor is used in a Flow, these bindings can be configured via the Python or YAML Flow syntax, so each Flow can use the Executor in its own way. However, the standard way of adding `@requests` in the Executor itself would still work just fine.
```